DAQ/test/generateDigiFromFragment.fcl

```diff
@@ -18,32 +18,47 @@ source : {
 physics : {
 
   producers : {
-    makeSD:
-    {
-      module_type: StrawAndCaloDigisFromFragments
-      diagLevel: 3
-      parseCAL: 1
-      parseTRK: 1
-
-      trkTag: "FragmentGenerator:offlinetrk"
-      caloTag: "FragmentGenerator:offlinecalo"
-    }
+      makeSD:
+      {
+	  module_type: StrawAndCaloDigisFromFragments
+	  diagLevel: 0
+	  parseCAL: 0
+	  parseTRK: 1
+	  
+	  trkTag  : "daq:trk"
+	  caloTag : "daq:calo"
+      }
+
+      CaloDigiFromShower:		
+      {
+	  module_type: StrawAndCaloDigisFromFragments
+	  diagLevel: 0
+	  parseCAL: 1
+	  parseTRK: 0
+	  
+	  trkTag  : "daq:trk"
+	  caloTag : "daq:calo"
+      }
```

Comment:
Same comment as above - can you use a predefined standard definition here?

Reply:
Done.